demo/demo-pojo/pojo-client/src/main/resources/microservice.yaml

```diff
@@ -23,6 +23,12 @@ servicecomb:
   service:
     registry:
       address: http://127.0.0.1:30100
+      instance:
+        healthCheck:
+          interval: 2
+        watch: false
+        empty:
```

Comment:
Please update document if there is new option introduced. 
Is there any other empty related options?

Reply:
Currently there are a lot of configurations not documented. I create a new issue to tracker this and will add docs after we have merged documents as [VOTE] done in email. 
https://issues.apache.org/jira/browse/SCB-594